ovh/resource_ovh_iploadbalancing_http_frontend.go

```diff
@@ -153,13 +159,14 @@ func resourceIpLoadbalancingHttpFrontendUpdate(d *schema.ResourceData, meta inte
 	}
 
 	frontend := &IpLoadbalancingHttpFrontend{
-		Port:          d.Get("port").(string),
-		Zone:          d.Get("zone").(string),
-		AllowedSource: allowedSources,
-		DedicatedIpFo: dedicatedIpFo,
-		Disabled:      getNilBoolPointerFromData(d, "disabled"),
-		Ssl:           getNilBoolPointerFromData(d, "ssl"),
-		DisplayName:   d.Get("display_name").(string),
+		Port:             d.Get("port").(string),
+		Zone:             d.Get("zone").(string),
+		AllowedSource:    allowedSources,
+		DedicatedIpFo:    dedicatedIpFo,
+		Disabled:         getNilBoolPointerFromData(d, "disabled"),
+		Ssl:              getNilBoolPointerFromData(d, "ssl"),
+		RedirectLocation: d.Get("redirect_location").(string),
```

Comment:
if we switch to *string, then we should use getNilStringPointerFromData(d, "redirect_localtion")